test/WebJobs.Script.Tests/ContentEndToEndTests.cs

```diff
@@ -19,9 +19,9 @@
 
 namespace Microsoft.Azure.WebJobs.Script.Tests.Content
 {
-    public class NodeTextPlain : ContentTests
+    public class NodeTextPlain : ContentTests<NodeContentFixture>
```

Comment:
Save a few (maybe lol) lines of code, didn't like to specify the content type over and over again.  Agree with your point on the ScriptHost startup costs, however they aren't incurred as the tests actually share the same fixture